wdqs/0.3.0/Dockerfile

```diff
@@ -24,7 +24,9 @@ ENV MEMORY=""\
     HOST="0.0.0.0"\
     WIKIBASE_HOST="wikibase.svc"\
     WDQS_HOST="wdqs.svc"\
-    WDQS_PORT="9999"
+    WDQS_PORT="9999"\
+    WDQS_ENTITY_NAMESPACES="120,122"\
+    WIKIBASE_SCHEME="http"
```

Comment:
This looks a little misplaces down here. This belongs to the host above, and would make a little more sense when placed in the line before.